### PR TITLE
Task/update bucket customizer dqa adjustments branch

### PR DESCRIPTION
**Pull request by alliuca**
**Description:** Updates main bucket branch with the latest B21.01 changes

**Ticket:** https://moderntribe.atlassian.net/browse/TEC-3696

### CODE REVIEW

File: src/Tribe/Customizer.php [hunk 1]
```diff
@@ -125,7 +125,7 @@ public function __construct() {
 		 * By default inline scripts will be printed on the `wp_enqueue_scripts` action, but other
 		 * plugins or later iterations might require inline styles to be printed on other actions.
 		 *
-		 * @since TBD
+		 * @since 4.12.15
```
Comment (tr1b0t) [inline src/Tribe/Customizer.php:128]: :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: f749ccc26a087aba4ab6ca5c1c3e3d4d</sub>

File: src/Tribe/Customizer.php [hunk 2]
```diff
@@ -433,7 +433,7 @@ public function widget_inline_style() {
 	/**
 	 * Print the CSS for the customizer using wp_add_inline_style
 	 *
-	 * @since TBD Added the `$force` parameter to force the print of the style inline.
+	 * @since 4.12.15 Added the `$force` parameter to force the print of the style inline.
```
Comment (tr1b0t) [inline src/Tribe/Customizer.php:436]: * :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :warning: Line exceeds 80 characters; contains 90 characters

<sub>hash: 3538874eb355aa89b9e7a60d88ec1aa8</sub>

File: src/Tribe/Customizer.php [hunk 3]
```diff
@@ -485,7 +485,7 @@ public function inline_style( $force = false ) {
 				/**
 				 * Fires before a style is, possibly, printed inline depending on the stylesheet.
 				 *
-				 * @since TBD
+				 * @since 4.12.15
```
Comment (tr1b0t) [inline src/Tribe/Customizer.php:488]: :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 4d38aa0d80a394fdb2e81f96396460ce</sub>

File: src/Tribe/Customizer.php [hunk 4]
```diff
@@ -583,7 +583,7 @@ public function register( WP_Customize_Manager $customizer ) {
 			 * Allows people to Register and de-register the method to register more Fields
 			 *
 			 * @since 4.4
-			 * @since TBD Add Customizer instance as a parameter.
+			 * @since 4.12.15 Add Customizer instance as a parameter.
```
Comment (tr1b0t) [inline src/Tribe/Customizer.php:586]: :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: ae4249caa84eb5f453cc0c656db978c6</sub>

File: src/Tribe/Log/Service_Provider.php [hunk 5]
```diff
@@ -151,7 +151,7 @@ public function add_logging_engine( array $logging_engines = [] ) {
 	/**
 	 * Enables logging in the service provider, if not already enabled.
 	 *
-	 * @since TBD
+	 * @since 4.12.15
```
Comment (tr1b0t) [inline src/Tribe/Log/Service_Provider.php:154]: :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: c5bce384583a3ba28d7fc42021a6d36a</sub>

File: src/Tribe/Log/Service_Provider.php [hunk 6]
```diff
@@ -164,7 +164,7 @@ public function enable() {
 	/**
 	 * Disables the logging functions.
 	 *
-	 * @since TBD
+	 * @since 4.12.15
```
Comment (tr1b0t) [inline src/Tribe/Log/Service_Provider.php:167]: :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: dcb92ce6070892036783bd78f76e5cda</sub>

File: src/Tribe/Main.php [hunk 7]
```diff
@@ -19,7 +19,7 @@ class Tribe__Main {
 	const OPTIONNAME          = 'tribe_events_calendar_options';
 	const OPTIONNAMENETWORK   = 'tribe_events_calendar_network_options';
 
-	const VERSION             = '4.12.16';
+	const VERSION             = '4.12.17';
```
Comment (tr1b0t) [inline src/Tribe/Main.php:22]: * :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :warning: Equals sign not aligned correctly; expected 1 space but found 13 spaces

<sub>hash: 072fff1936013fdf99739de16aa30c0b</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 7 error(s)
:warning: 2 warning(s)

**Review by lucatume — APPROVED** [review]

**Pull request merged**